src/lib.rs

```diff
@@ -158,7 +158,8 @@
 //! ```
 
 #![doc(html_logo_url = "https://www.rust-lang.org/logos/rust-logo-128x128-blk-v2.png",
-       html_favicon_url = "https://www.rust-lang.org/favicon.ico")]
+      html_favicon_url = "https://www.rust-lang.org/favicon.ico",
+      html_root_url = "https://docs.rs/semver/0.9.0")]
```

Comment:
Is there a reason this needs to be 0.9.0 instead of just linking to `https://docs.rs/semver/`? I'm new to adding these things, so I'm probably missing something.

Reply:
I added the url under the guidelines from #134  
It writes: "https://docs.rs/semver/MAJOR.MINOR.PATCH"
I will recommit it if you think the later(https://docs.rs/semver/) is better.

Reply:
So, I just discovered https://github.com/steveklabnik/semver-parser/pull/26#discussion_r147753088

the guideline changed!

Would you mind fixing it to remove this bit? Thanks again 😄

Reply:
ok, I will change it.
by the way, should I recommit all my commits based on newer Rustfmted master from #147 since this commit was merged?